.changeset/tall-onions-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': major
+---
+
+Implements complete redesign of button component. For more information on how to upgrade to version 11, see our upgrade guide.
```

Comment:
Can we make this a little more detailed? Just a cliff's notes version like:

```
- Redesign's the button component! 
- Improves accessibility of buttons for visually-impaired users.
- Adds support for right-aligned glyphs.
- Renames glyph prop to `leftGlyph`.
```

That's not comprehensive, but folks should be able to get the main changes from the changelog still.